config.json

```diff
@@ -4,11 +4,11 @@
   },
   "dnsLess": {
     "isActive": true,
-    "publicUrl": "http://localhost:3000"
+    "publicUrl": "https://my-computer.rec.la"
```

Comment:
Can you use another config, like adding it to `configs/dockerized-config.json`? as this one is used for running it natively?